misk/src/test/kotlin/misk/web/RequestTypesTest.kt

```diff
@@ -37,5 +37,8 @@ internal class RequestTypesTest {
     // Check repeated types
     assertThat(shipmentType.fields).contains(Field("notes", "String", true))
     assertThat(warehouseType.fields).contains(Field("alternates", Warehouse::class.qualifiedName!!, true))
+
+    // Check enum types
+    assertThat(shipmentType.fields).contains(Field("status", "Enum<${Shipment.State::class.qualifiedName!!},${Shipment.State.values().joinToString(",")}>", false))
```

Comment:
/nit how about we actually assert the full values rather than programatically calculating it? I prefer that as being more assertive than having to reason about what the expression resolves to